queries/earth.jinja2

```diff
@@ -1,78 +1,99 @@
 {% if zoom < 9 %}
 
 SELECT
-    NULL AS name,
-    'earth' AS kind,
-    NULL AS label_placement,
+    '' AS name,
+    sum(way_area)::bigint AS area,
+    mz_calculate_output_earth(t.*) AS mz_properties,
     'naturalearthdata.com' AS source,
     {% filter geometry %}st_union(the_geom){% endfilter %} AS __geometry__,
-    max(gid) AS __id__
+    max(gid) AS __id__,
+    NULL AS label_placement,
+    NULL AS boundary
```

Comment:
I think we prefer to do the addition of extra parameters (`label_placement`, `boundary`) via YAML now, to avoid having to do `NULL AS ...` in the other queries.